llvm/tools/llvm-mc/Disassembler.cpp

```diff
@@ -169,7 +169,7 @@ int Disassembler::disassemble(const Target &T, const std::string &Triple,
   }
 
   // Set up initial section manually here
-  Streamer.initSections(false, STI);
+  Streamer.switchSectionNoPrint(Ctx.getObjectFileInfo()->getTextSection());
```

Comment:
Not sure why should init sections / switch to text at all. What would happen if we just remove this line?

NB
There is a similar place in llvm-ml, might be worth fixing as well.

Reply:
I investigated further. This call is only needed by AIX XCOFF https://reviews.llvm.org/D95518 .

```
echo 7c4303a6 | rrllvm-mc --cdis --hex --triple=powerpc-aix-ibm-xcoff
```

I deleted this statement and adjusted MCAsmStreamer.